pkg/llm-d-inference-sim/config_test.go

```diff
@@ -46,6 +45,33 @@ func createSimConfig(args []string) (*configuration, error) {
 	return s.config, nil
 }
 
+func createDefaultBasicConfig(model string) *configuration {
+	c := newConfig()
+
+	c.Model = model
+	c.ServedModelNames = []string{c.Model}
+	c.MaxNumSeqs = 5
+	c.MaxLoras = 1
+	c.MaxCPULoras = 1
+	c.TimeToFirstToken = 2000
+	c.InterTokenLatency = 1000
+	c.KVCacheTransferLatency = 100
+	c.Seed = 100100100
+	c.LoraModules = []loraModule{}
+
+	return c
+}
+
+func createDefaultConfig(model string) *configuration {
+	c := createDefaultBasicConfig(model)
+
+	// parameters special to config.yaml
+	c.MaxLoras = 2
+	c.MaxCPULoras = 5
+
+	return c
+}
+
```

Comment:
I think this is confusing, and one function is enough. There is only one case for createDefaultBasicConfig, and we can just update the lora parameters in the test.